plugins/inputs/aerospike/aerospike.go

```
@@ -54,7 +57,11 @@ func (a *Aerospike) Gather(acc telegraf.Accumulator) error {
 	return errChan.Error()
 }
 
-func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) error {
+func (a *Aerospike) gatherServer(rawurl string, acc telegraf.Accumulator) error {
+	rawurl = "aero://" + rawurl
+	urlInfo, err := url.Parse(rawurl)
```

you should return this error if it's not nil